.github/patches/extensions/httpfs/httpfs_loader_and_timestamp.patch

```diff
@@ -206,9 +206,14 @@ index 686162f..cb55421 100644
 -	tm.tm_sec = parse_result.data[5];
 -	tm.tm_isdst = 0;
 -	result = mktime(&tm);
-+	auto date = Date::FromDate(parse_result.data[0] - 1900, parse_result.data[1] - 1, parse_result.data[2]);
++	date_t date;
++	if (!Date::TryFromDate(parse_result.data[0] - 1900, parse_result.data[1] - 1, parse_result.data[2], date)) {
```

Comment:
I think the `- 1900` and `- 1` here are wrong. [mktime](https://linux.die.net/man/3/mktime) has the following semantics for year/month:

> tm_year
> The number of years since 1900.

> tm_mon
> The number of months since January, in the range 0 to 11.
 
Which is why the `- 1900` and `-1` were there before.